lib/rubocop/cop/layout/access_modifier_indentation.rb

```diff
@@ -66,6 +66,7 @@ def autocorrect(node)
 
         def check_body(body, node)
           return if body.nil? # Empty class etc.
+          return if body.module_type? || body.class_type? || body.sclass_type?
```

Comment:
Seems to me that a better way to handle this would be to check that `body` is a begin node and just go over it's top-level nodes, instead of recursing into them with `each_child_node`. It's always better to whitelist than to blacklist. :-)

Reply:
Like this?